nsv13/code/modules/power/stormdrive.dm

```diff
@@ -439,13 +319,36 @@ Takes  plasma and outputs superheated plasma and a shitload of radiation.
 	handle_heat()
 	update_icon()
 	radiation_pulse(src, heat, 2)
+	ambient_temp_bleed()
+
+/obj/machinery/power/stormdrive_reactor/proc/grav_pull() //HUNGRY!
+	for(var/obj/O in orange(6, src))
+		if(!O.anchored)
+			step_towards(O,src)
+	for(var/mob/living/M in orange(6, src))
+		if(!M.mob_negates_gravity())
+			step_towards(M,src)
+			M.Knockdown(40) //Knockdown prey so it can't get away!
+
+/obj/machinery/power/stormdrive_reactor/proc/ambient_temp_bleed()
+	var/turf/open/L = loc
+	if(!istype(L) || !(L.air))
+		return
+	var/datum/gas_mixture/env = L.return_air()
+//	var/heat_capacity = env.heat_capacity()
+	var/heat_kelvin = heat + 273.15
+	if(env.temperature <= heat_kelvin)
+		var/delta_env = heat_kelvin - env.temperature
+		env.temperature += delta_env / 2
+		air_update_turf()
 
 /obj/machinery/power/stormdrive_reactor/proc/can_cool()
 	if(heat > REACTOR_HEAT_NORMAL+10) //Only start melting the rods if theyre running it hot. We have a "safe" mode which doesn't need you to check in on the reactor at all.
 		rod_integrity -= input_power_modifier/60 //Assuming youre running it at hot, rods will melt every 30 minutes.
 		if(rod_integrity < 0)
 			rod_integrity = 0
 			send_alert("DANGER: Primary control rods have failed!")
+			message_admins("Stormdrive control rods are at 0% integrity.")
```

Comment:
Do we not want players to know that control rods have failed?

Reply:
They already do get notified through send_alert().
This is just a direct to admins message.